### DISCUSSION TITLE
[RELEASE] City

### DISCUSSION THREAD

**Discussion by Paindeluxe** [nzp-team/nzportable#213] [Map Releases]
# Info about the Map

After the failed experiment named [Tag der Untoten](https://github.com/nzp-team/nzportable/discussions/199)
I decided to make a better map and basically a sequel to [Alleyway](https://github.com/nzp-team/nzportable/discussions/173) called City.

This Map features a Day and Night version.

It's rather long and it contains Random Perks to hopefully spice up gameplay a bit.

This most likely runs **smoothly** on the _PSP_ as I have tested it before near the end of Development.

# Preview
This is the map at Daytime.
![fte-20230224150950-0](https://user-images.githubusercontent.com/104042185/221200550-ea461144-aad7-4523-9efe-d81db83b722d.png)

And this is the map during the Night.
![fte-20230224151034-0](https://user-images.githubusercontent.com/104042185/221200579-48c383a5-7b42-4086-ac0c-05264bdb48cd.png)

# Update Log

Please use the download button down below to download the latest version!

- V1: ([Release](https://github.com/nzp-team/nzportable/files/10825460/City.zip)) Made the Map.
Please download the [Pathfinding Improvements](https://archive.org/download/city-updated-waypoints/city%20updated%20waypoints.zip) which have been made by [M0THRAK](https://github.com/nzp-team/nzportable/discussions/213#discussioncomment-16750856) alongside the Download from V1.
- V2: (Not made yet!) Will feature changed textures to the Night Version and maybe some other Stuff like opening up the map some more it will also feature zombie spawns in the Main area.

# [Download](https://github.com/nzp-team/nzportable/files/10825460/City.zip)

**Comment by PrimiseRichtofen** [comment]
![image](https://github.com/user-attachments/assets/afae4436-3a0d-46f9-a326-6c93ffcf4efe)

**Reply by Paindeluxe** [reply]
Whuh… I am both in shock and awe at this. Thank you for playing my map! :D

**Reply by PrimiseRichtofen** [reply]
Your welcome :D

**Reply by PrimiseRichtofen** [reply]
I might try and go to fifty I just needa try to get a good perk rng

**Reply by PrimiseRichtofen** [reply]
I'll even do a starting room challenge if you updated the map to have a wall buy in the starting room also it lags alot on the web version maybe because and the zombies will pile into one another in spawn when your in the main area

**Reply by Paindeluxe** [reply]
I'll make sure to get back into NZP mapping as soon as possible. I'm just incredibly happy that my Maps still get played, the lack of feedback kind of made me run out of passion for this (:

**Reply by ghost** [reply]
100% I'm just getting into NZP after fixing my vita so all maps are fresh to me, looking forward to of you make anymore maps

**Comment by PrimiseRichtofen** [comment]
[Screen recording 2025-05-26 6.56.42 PM.webm](https://github.com/user-attachments/assets/83e82338-e5ab-44cf-a202-835803bc8ac1)

**Reply by PrimiseRichtofen** [reply]
the zombies pilling up makes it lag like a mother fucker other then that good map.

**Comment by M0THRAK** [comment]
I've redone the waypoints on this map if anyone is interested in using them for whatever reason (this fixes pathfinding on this map), you can download them [here](https://archive.org/download/city-updated-waypoints/city%20updated%20waypoints.zip).

**Reply by Paindeluxe** [reply]
Is it ok if I link your Version to my Message of the Map Release?
I find it very nice that you did this even though I abandoned the Map

**Reply by M0THRAK** [reply]
Go right ahead!
